ee/allowedcmd/cmd.go

```diff
@@ -31,15 +34,15 @@ func validatedCommand(ctx context.Context, knownPath string, arg ...string) (*ex
 	// We expect to know the exact location for allowlisted commands on all
 	// OSes except for a few Linux distros.
 	if !allowSearchPath() {
-		return nil, fmt.Errorf("not found: %s", knownPath)
+		return nil, fmt.Errorf("%w: %s",ErrCommandNotFound, knownPath)
```

Comment:
Tiny nitpick -- I think the spacing is off here --

```suggestion
		return nil, fmt.Errorf("%w: %s", ErrCommandNotFound, knownPath)
```